st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -305,6 +305,12 @@
                 'type': 'boolean',
                 'default': False
             },
+            'verify_ssl_cert': {
+                'description': 'SSL verification is not supported.',
+                'type': 'boolean',
+                'default': False,
+                'immutable': True
```

Comment:
Agreed with others - we should default to verifying the cert.

By default we should use system ca bundle and bundle shipped with requests which included most common CAs so this should cover the common use case.

Supporting custom CA bundle would also be nice (e.g. for internal infrastructure), but for now users can just disable SSL verification in case they use a custom CA.